atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/RowsColumnRangeBatchRequestsTest.java

```diff
@@ -134,9 +134,9 @@ private static void assertPartitionsHaveCorrectSize(
             int actualPartitionSize = RowsColumnRangeBatchRequests.getAllRowsInOrder(partitions.get(i))
                     .size();
             if (i < partitions.size() - 1) {
-                Assert.assertEquals(expectedSize, actualPartitionSize);
+                assertThat(actualPartitionSize).isEqualTo(expectedSize);
             } else {
-                Assert.assertTrue(actualPartitionSize <= expectedSize);
+                assertThat(actualPartitionSize <= expectedSize).isTrue();
```

Comment:
isLessThanOrEqualTo

Reply:
Same here, it will eventually make it to `isLessThanOrEqualTo` :-)